src/main/java/com/Saurabh/GitHubAiAPI/StringManipulation.java

```diff
@@ -45,4 +49,11 @@ public static String findVowels(String input) {
         }
         return vowels.toString();
     }
```

Comment:
The isPalindrome method is case-insensitive. If case-sensitive palindrome checking is required in the future, consider adding a parameter to the method to control this behavior.